src/main/java/seedu/anilist/logic/LogicManager.java

```diff
@@ -28,7 +27,7 @@
 public class LogicManager implements Logic {
     public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: ";
 
-    private Optional<Command> lastCmd;
+    private Command lastCmd;
```

Comment:
sorry why is the optional removed?

Reply:
yeah why did you change this?

Reply:
I think Optional was not needed at all. The code functions even without Optional so I think that its pointless using Optional when it doesn't serve any real purpose. What's the reason Optional was used here in the first place?